gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-healthcheck/src/test/java/io/gravitee/gateway/services/healthcheck/http/AbstractManagedEndpointRuleHandlerTest.java

```diff
@@ -242,9 +234,23 @@ public void handle(EndpointStatus status) {
         async.awaitSuccess();
     }
 
-    private Endpoint createEndpoint() {
-        HttpEndpoint aDefault = new HttpEndpoint("default", "http://localhost:" + wireMockRule.port());
+    private Endpoint createEndpoint(String targetPath) {
+        HttpEndpoint aDefault = new HttpEndpoint(
+            "default",
+            "http://localhost:" + wireMockRule.port() + (targetPath != null ? "/" + targetPath : "")
```

Comment:
Because you already use a "/" line 201, no need to add a new "/" here
```suggestion
            "http://localhost:" + wireMockRule.port() + (targetPath != null ? targetPath : "")
```